rcljava/cmake/add_source_jar.cmake

```diff
@@ -0,0 +1,194 @@
+function(add_source_jar _TARGET_NAME)
```

Comment:
It'd be nice to document this function. [Here's an example](https://github.com/ament/ament_lint/blob/d17140b716eb11f7cc8b3d686104ed2167ce9818/ament_cmake_cpplint/cmake/ament_cpplint.cmake#L15-L35).